src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -655,4 +656,30 @@ BOOST_AUTO_TEST_CASE(MishFunctionTest)
   CheckDerivativeCorrect<MishFunction>(desiredActivations,
                                         desiredDerivatives);
 }
+
+BOOST_AUTO_TEST_CASE(GELUFunctionTest)
+{
+  // Calculated using torch.nn.gelu().
+  const arma::colvec desiredActivations("-0.04540230591222 \
```

Comment:
Where do this values come from, manually calculated?

Reply:
Yes, I manually calculated as well I verified it using pytorch https://colab.research.google.com/drive/1rtsrVD2L-we8eXzeiu6QhC2gp96VNyxO